app/views/spree/shared/_nav_bar.html.erb

```diff
@@ -27,9 +27,90 @@
         </div>
       </div>
     </li>
-  <% end %>
+  <%# end %>
   <li id="link-to-cart">
-    <%= render 'spree/shared/cart', class: 'd-inline-block cart-icon', size: 36, item_count: 0 %>
+    <%#= render 'spree/shared/cart', class: 'd-inline-block cart-icon', size: 36, item_count: 0 %>
   </li>
-  <%= render partial: 'spree/shared/internationalization_options' %>
-</ul>
+  <%#= render partial: 'spree/shared/internationalization_options' %>
+</ul> -->
+<%# i have missed  id="link-to-account" because of design issue %>
```

Comment:
please discuss about this point with Umer. We need this div and id at any cost

Reply:
We have a separate ticket for that. Will handle in next sprint by Yaseen